docs/source/developers/python.rst

```diff
@@ -138,10 +138,16 @@ You should now see
 Using Conda
 ~~~~~~~~~~~
 
+.. note::
+
+   As of April 2019, using conda to build Arrow on macOS is complicated by the
```

Comment:
I object to this statement. I still consider using `conda` as the best method as we are also maintainers of the toolchain there.

Reply:
The installation instructions as currently written need to be revised one way or another, either to recommend installing that unofficial macOS SDK or to advise against conda on macOS. I chose the latter, but I'm happy to reconsider if you/someone will advocate for the former. The profile of the GH user hosting the SDKs doesn't inspire confidence in their legitimacy, so I was wary of making a formal recommendation in our docs.

If we maintain the conda toolchain, how hard is it for us to fix it so that old SDK isn't required? Easy enough that we can just do it now and drop this statement? Or should we add this note in the meantime?

Reply:
If You want to use the compilers shipped by conda-forge and the dependencies from conda forge then, then indeed an older SDK is required, but this is not true for the system compiler or for clang shipped by homebrew. I usually compile arrow with llvm from brew, the latest sdk and dependencies from CF.

| Compiler         | Dependencies | SDK   |
|------------------|--------------|-------|
| CF Clang*               | CF           | 10.9  |
| Apple LLVM/Clang | CF           | 10.14 |
| Brew LLVM/Clang  | CF           | 10.14 |

* https://docs.conda.io/projects/conda-build/en/latest/resources/compiler-tools.html